gramps/gen/utils/place.py

```diff
@@ -59,7 +60,7 @@
 West = West.replace("1", " ").strip()
 
 # build dictionary with translation en to local language
-translate_en_loc = {}
+translate_en_loc: dict[str, str] = {}
```

Comment:
You need `from __future__ import annotations` here for `dict`, I believe.

Reply:
PR updated. Thanks.